src/controllers/sendResetToken.js

```diff
@@ -0,0 +1,109 @@
+import { randomBytes } from 'crypto';
+import user from '../db/models';
+import sendgrid from '../middlewares/helper';
+
+const { User } = user;
+
+/**
+  * @description class representing reset user password
+  *
+  * @class SendResettoken
+  */
+class SendResetToken {
+  /**
+    * @description - This method is responsible for creating creating password reset token
+    *
+    * @static
+    * @param {object} request - Request sent to the router
+    * @param {object} response - Response sent from the controller
+    *
+    * @returns {object} - object representing response message
+    *
+    * @memberof UserHandler
```

Comment:
`@memberof UserHandler`?

Reply:
Thanks for pointing that out. it has been resolved.